metricbeat/module/jolokia/_meta/fields.yml

```diff
@@ -11,3 +11,31 @@
       description: >
           jolokia contains metrics exposed via jolokia agent
       fields:
+        - name: agent.version
```

Comment:
Not directly related to this PR but I just realise now that this at one stage should become an ECS field and Metricbeat in this case is the observer. @jsoriano is this correct?

Reply:
Do you mean setting `observer.version` to this field and `observer.type` to `jolokia`? I guess it could make sense, but yeah, nothing to be done on this PR about this.

Reply:
Yesish. This `agent.version` would end up on the top level an `observer.version` would be the current beat version. It's a more general issue we have but just jumped in my face here :-D